airflow/cli/commands/role_command.py

```diff
@@ -41,3 +39,259 @@ def roles_create(args):
+    ("LogModelView", "can_list"): [("Log", "can_read")],  # views.py
+    ("MenuApi", "can_get"): [],  # FAB menu.py
+    ("PermissionModelView", "can_list"): [("Permission", "can_read")],  # FAB
+    ("PermissionViewModelView", "can_list"): [("PermissionView", "can_read")],  # FAB
+    ("PoolModelView", "can_add"): [("Pool", "can_create")],  # views.py
+    ("PoolModelView", "can_delete"): [("Pool", "can_delete")],  # views.py
+    ("PoolModelView", "can_edit"): [("Pool", "can_edit")],  # views.py
+    ("PoolModelView", "can_list"): [("Pool", "can_read")],  # views.py
+    ("PoolModelView", "muldelete"): [("Pool", "can_delete")],  # views.py
+    ("ResetMyPasswordView", "can_this_form_get"): [],  # FAB
+    ("ResetMyPasswordView", "can_this_form_post"): [],  # FAB
+    ("ResetPasswordView", "can_this_form_get"): [],  # FAB
+    ("ResetPasswordView", "can_this_form_post"): [],  # FAB
+    ("RoleModelView", "can_add"): [("Role", "can_create")],  # FAB
+    ("RoleModelView", "can_delete"): [("Role", "can_delete")],  # FAB
+    ("RoleModelView", "can_download"): [("Role", "can_read")],  # FAB
+    ("RoleModelView", "can_edit"): [("Role", "can_edit")],  # FAB
+    ("RoleModelView", "can_list"): [("Role", "can_read")],  # FAB
+    ("RoleModelView", "can_show"): [("Role", "can_read")],  # FAB
+    ("RoleModelView", "copyrole"): [("Role", "can_create")],  # FAB
+    ("SlaMissModelView", "can_list"): [("SlaMiss", "can_read")],  # views.py
+    ("Task", "can_edit"): [("Task", "can_edit")],  # task_endpoint.py
+    ("Task", "can_read"): [("Task", "can_read")],  # task_endpoint.py
+    ("TaskInstanceModelView", "can_list"): [("TaskInstance", "can_read")],  # views.py
+    ("TaskInstanceModelView", "clear"): [],  # views.py
+    ("TaskInstanceModelView", "set_failed"): [("TaskInstance", "can_edit")],  # views.py
+    ("TaskInstanceModelView", "set_retry"): [("TaskInstance", "can_edit")],  # views.py
+    ("TaskInstanceModelView", "set_running"): [("TaskInstance", "can_edit")],  # views.py
+    ("TaskInstanceModelView", "set_success"): [("TaskInstance", "can_edit")],  # views.py
+    ("TaskRescheduleModelView", "can_list"): [("TaskReschedule", "can_read")],  # views.py
+    ("UserDBModelView", "can_add"): [("UserDB", "can_create")],  # FAB
+    ("UserDBModelView", "can_delete"): [("UserDB", "can_delete")],  # FAB
+    ("UserDBModelView", "can_download"): [],  # FAB
+    ("UserDBModelView", "can_edit"): [("UserDB", "can_edit")],  # FAB
+    ("UserDBModelView", "can_list"): [("UserDB", "can_read")],  # FAB
+    ("UserDBModelView", "can_show"): [],  # FAB
+    ("UserDBModelView", "can_userinfo"): [],  # FAB
+    ("UserDBModelView", "resetmypassword"): [],  # FAB
+    ("UserDBModelView", "resetpasswords"): [],  # FAB
+    ("UserDBModelView", "userinfoedit"): [],  # FAB
+    ("UserInfoEditView", "can_this_form_get"): [],  # FAB
+    ("UserInfoEditView", "can_this_form_post"): [],  # FAB
+    ("UserStatsChartView", "can_chart"): [],  # FAB
```

Comment:
These are FAB built in permissions -- should we leave them alone, or have we subclassed the view to change the resource name?